core/domain/stats_services.py

```diff
@@ -77,7 +91,12 @@ def get_versions_for_exploration_stats(exploration_id):
 def get_exploration_stats(exploration_id, exploration_version):
     """Returns a dict with state statistics for the given exploration id.
 
-    Note that exploration_version should be a string.
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: str. The version of the exploration.
```

Comment:
Just to double-check: how did you verify that this is a str and not an int? The version is sometimes represented as str and sometimes as int so I wanted to understand how you figured the type out. If it's a str, are there any non-integer values it can take?